content/geoip/release-notes/2023.mdx

```diff
@@ -22,7 +22,7 @@ The `Satellite` connection type value will appear in the following databases:
 Users of the following client APIs will need to update their API to the latest
 version to avoid breaking changes with the new value:
 
-- [Update Java API to version 4.1.0](https://central.sonatype.com/search?q=g:com.maxmind.geoip2%20%20a:geoip2&smo=true)
+- [Update Java API to version 4.1.0](https://central.sonatype.com/artifact/com.maxmind.geoip2/geoip2/4.1.0)
```

Comment:
We can probably drop the `4.1.0` from the URL unless this exact version is needed.

Reply:
leaving it off defaults to the most current version, is that correct?

Reply:
Yes, please see the above comment: https://github.com/maxmind/dev-site/pull/1027#discussion_r1307431947.

You can also test it by going to https://central.sonatype.com/artifact/com.maxmind.geoip2/geoip2.

Reply:
sorry, dallas. i saw the correction above, but not the comment. i don't know if the browser didn't load the comment or if i just missed it.